packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -415,16 +405,16 @@ export class Outbox {
 	 * @remarks - If chunking happens, a side effect here is that 1 or more chunks are queued immediately for sending in next JS turn.
 	 *
 	 * @param batch - Raw or Grouped batch to consider for compression/chunking
-	 * @returns Either (A) the original batch, (B) a compressed batch (same length as original),
-	 * or (C) a batch containing the last chunk (plus empty placeholders from compression if applicable).
+	 * @returns Either (A) the original batch or (B) a compressed batch (same length as original),
```

Comment:
You're not changing anything about chunking, so let's leave (C) here.

```suggestion
	 * @returns Either (A) the original batch, (B) a compressed batch,
	 * or (C) a batch containing the last chunk.
```

Also, I'd mention in the main comment that only single-message batches are supported (and can we update the type?)

Reply:
Applied the change and I'll create the item to follow up on the type change